clients/fossid-webapp/src/main/kotlin/model/status/ScanState.kt

```diff
@@ -22,13 +22,40 @@ package org.ossreviewtoolkit.clients.fossid.model.status
 import com.fasterxml.jackson.annotation.JsonProperty
 
 enum class ScanState {
```

Comment:
For me as a non-FossID expert, it would be helpful to give some context in the class comment as well, e.g. something like "An enumeration with the status codes reported by FossID when the state of a scan is queried(?)".